src/dials/report/analysis.py

```diff
@@ -285,6 +286,16 @@ def table_1_stats(
         "CC half": "cc_one_half",
         "Total unique": "n_uniq",
     }
+    extra_key_to_var = {}
+    if merging_statistics.r_split:
+        extra_key_to_var.update(
+            {
+                "Rsplit": {
+                    "overall": "r_split",
+                    "binned": "r_split_binned",
+                }
+            }
+        )
```

Comment:
Am I missing something here why we need a separate key-var dictionary for the additional Rsplit statistic?

Reply:
Yes this is needed as the other dictionary is looped over when inspecting the attributes of the merging stats bin objects, but the Rsplit is stored separately so needs a different lookup mechanism.